### PR TITLE
Dark Mode: Notifications

### PR DESCRIPTION
**Pull request by frosty**
Refs #12320. This PR implements dark mode fixes for the Notifications section of the app.

dark-mode-notifications

**To test:**

* Build and run in light and dark mode, and iOS 12
* Check for anything that doesn't look right

Update release notes:

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

cc @mattmiklic

### CODE REVIEW

File: WordPress/Classes/Extensions/Colors and Styles/UIColor+MurielColors.swift [hunk 1]
```diff
@@ -258,6 +258,24 @@ extension UIColor {
...
     static var primaryButtonDownBackground = muriel(color: .accent, .shade80)
     static var primaryButtonDownBorder = muriel(color: .accent, .shade90)
+
+    static var secondaryButtonBackground: UIColor {
+        #if XCODE11
+            if #available(iOS 13, *) {
+                return UIColor(light: .white, dark: .systemGray5)
+            }
+        #endif
+        return UIColor(light: .white, dark: .neutral(.shade80))
```
Comment (etoledom) [inline WordPress/Classes/Extensions/Colors and Styles/UIColor+MurielColors.swift]: Based on https://github.com/wordpress-mobile/WordPress-iOS/pull/12386#discussion_r317383172 , it probably also apply here?

**Comment by etoledom** [comment]
Should the text on `Go to Reader` button be white?

I saw it on back also on My Site -> Post / Pages no-result views.

Simulator Screen Shot - iPhone Xs - 2019-08-26 at 11 29 32

**Comment by frosty** [comment]
It should! I think this will be resolved by the NUX PR, as I changed the main appearance for them there: https://github.com/wordpress-mobile/WordPress-iOS/pull/12386/files#diff-75f4435634db3b82756c8c6c2f0b231bR471

**Comment by mattmiklic** [comment]
Looking good overall! 

For the spam/trash/like/edit icons and text, could we switch those to use `listIcon` and `textSubtle` colors, to match what we're using elsewhere?

**Pull request merged**